tests/hpc/hpc_master_backup.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: HPC master - backup master node
+#    This test is aiming at comprehensive HPC testing. That means
+#    that most components of the HPC product should be installed and
+#    configured. At the same time it is expected that the HPC cluster
+#    set-up is fairly complex, so that NFS shares are mounted, required
+#    database(s) are ready to be used, artificial users are added etc.
+# Maintainer: Sebastian Chlad <sebastian.chlad@suse.com>
+
+use base "hpcbase";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+
+sub mount_nfs {
```

Comment:
It would be advisable to move this to lib/hpcbase.pm like your TODO comment suggests. This may also help your resolve that travis complaints about duplicated code.

Reply:
done